mathenger-api/src/test/java/com/example/mathengerapi/integrations/dspace/events/DSpaceMessageHandlerTest.java

```diff
@@ -81,6 +85,152 @@ void shouldHandleAllCommunitiesCommand() {
+        List<Collection> collections = List.of(
+                createCollection(UUID.fromString("13d93fd6-5eb0-4952-9bbb-4b6e417e1160"), "1st collection", "", ""),
+                createCollection(UUID.fromString("23d93fd6-5eb0-4952-9bbb-4b6e417e1160"), "2nd collection", "2nd introductoryText", "2nd shortDescription")
+        );
+        when(dSpaceClient.getCollections()).thenReturn(collections);
+        List<Item> items = List.of(
+                createItem(UUID.fromString("33d93fd6-5eb0-4952-9bbb-4b6e417e1160"), "1st item"),
+                createItem(UUID.fromString("43d93fd6-5eb0-4952-9bbb-4b6e417e1160"), "2nd item")
+        );
+        var expectedMessage = "About this collection:\n\n"
+                + "2nd introductoryText\n"
+                + "2nd shortDescription\n" + "\n"
+                +"Here you can view a list of publications in the selected collection:\n\n"
+                + "1st item /id:33d93fd6-5eb0-4952-9bbb-4b6e417e1160\n"
+                + "2nd item /id:43d93fd6-5eb0-4952-9bbb-4b6e417e1160\n\n"
+                + "If you want to read one, use the command “/publication_{id}";
+        when(dSpaceClient.getItemsOfCollection(UUID.fromString("23d93fd6-5eb0-4952-9bbb-4b6e417e1160"))).thenReturn(items);
+
+        send("/colpublications_23d93fd6-5eb0-4952-9bbb-4b6e417e1160");
+
+        await().atMost(AWAITING_TIMEOUT, TimeUnit.SECONDS).untilAsserted(() -> {
+            verify(messageService).sendMessage(eq(botId), textEquals(expectedMessage), eq(CHAT_ID));
+            verify(dSpaceClient, times(1)).getItemsOfCollection(UUID.fromString("23d93fd6-5eb0-4952-9bbb-4b6e417e1160"));
+        });
+    }
+
+
+    @Test
+    void shouldHandleHelpCommand() {
+        Long botId = botInfoHolder.getBotAccount().getId();
+        var expectedMessage = "/community - display a list of all communities\n" +
+                "/community_{id} - display a list of collections that are in the selected community\n" +
+                "/colpublications_{id} - display the list of all works in the collection\n" +
+                "/help - display a list of all commands";
+
+
+        send("/help");
+
+        await().atMost(AWAITING_TIMEOUT, TimeUnit.SECONDS).untilAsserted(() -> {
+            verify(messageService).sendMessage(eq(botId), textEquals(expectedMessage), eq(CHAT_ID));
+
+        });
+    }
+
+    @Test
+    void shouldErrorCommand() {
+        Long botId = botInfoHolder.getBotAccount().getId();
+        var expectedMessage = "Sorry, I can’t understand errorCommand.\n" +
+                "You can talk to me with these commands:\n" +
+                "\n" +
+                "/community - display a list of all communities\n" +
+                "/community_{id} - display a list of collections that are in the selected community\n" +
+                "/colpublications_{id} - display the list of all works in the collection\n" +
+                "/help - display a list of all commands";
```

Comment:
Pls extract commands list to constant COMMANDS just like I did earlier in the commands handler